src/buffer/out/textBuffer.cpp

```diff
@@ -2088,26 +2076,26 @@ std::wstring TextBuffer::GetPlainText(const til::point& start, const til::point&
 // Routine Description:
 // - Generates a CF_HTML compliant structure based on the passed in text and color data
 // Arguments:
-// - rows - the text and color data we will format & encapsulate
+// - rows - the text and attribute data we will format & encapsulate
 // - backgroundColor - default background color for characters, also used in padding
 // - fontHeightPoints - the unscaled font height
 // - fontFaceName - the name of the font used
+// - isIntenseBold - true if being intense is treated as being bold
 // Return Value:
 // - string containing the generated HTML
-std::string TextBuffer::GenHTML(const TextAndColor& rows,
+std::string TextBuffer::GenHTML(const TextAndAttribute& rows,
                                 const int fontHeightPoints,
                                 const std::wstring_view fontFaceName,
-                                const COLORREF backgroundColor)
+                                const COLORREF backgroundColor,
+                                const bool isIntenseBold)
 {
     try
     {
         std::ostringstream htmlBuilder;

```

Comment:
BTW: If you ever want to, on my end, you're more than welcome to replace the `std::ostringstream` with a `std::string` and use regular `.append()` or `+=` and `fmt::format_to(FMT_COMPILE(...), ...)` instead. 😄 `sstream` classes have nasty overhead, both for binary size and runtime performance. And they aren't even particularly powerful.

Reply:
Let's see if we can take the alternative approach we're discussing above. If not, I'll come back to this 🙂